tests/AnalyzeSpec.hs

```diff
@@ -3640,7 +3640,7 @@ spec = describe "analyze" $ do
         (succeeds-when (> x 1000) { 'except: [] })
         (fails-when    (<= x 0)   { 'except: [] })
       |]
-      "(defun test:bool (x:integer) (enforce (> x 0)))"
+      "(defun test:bool (x:integer) (enforce (> x 0) \"\"))"
 
   describe "scope-checking interfaces" $ do
     res <- runIO $ checkInterface [text|
```

Comment:
FYI @bts @joelburget we fixed a bug in the typechecker that affected alot of analysis code.